sdk/cosmosdb/cosmos/test/tsconfig.json

```diff
@@ -9,8 +9,10 @@
     "allowSyntheticDefaultImports": true,
     "preserveConstEnums": true,
     "removeComments": false,
-    "target": "es6",
-    "lib": ["es2019"],
+    "target": "ES2022",
```

Comment:
We are not in a position currently to upgrade the ES target version. I think this change was made only to add support for `AggregateError`. 

Let's revert this and the lib to the previous version only.

Reply:
I've reverted the typescript change for now to avoid the debate. Not critical for the PR.